__exekutir__/inventory/host_vars/kommandir.yml

```diff
@@ -87,4 +87,6 @@ git_cache_args:
     - repo: "https://github.com/python-bugzilla/python-bugzilla.git"
       recursive: False
       cachepath: "python-bugzilla"
+      # Do not include a 'depth' argument, git needs the full history to be 
+      # able to locate specific commits/branches/tags referenced by 'version'
```

Comment:
No documentation _in this module_. There is no explanation that `git_cache_args` will be passed to the `git` module, no explanation of the arguments, no explanation of anything. Whether one would be useful or not is beyond the scope of this discussion. My point is: if there were documentation in this file, and that documentation included the `depth` option, then a "this will be ignored if you also give a version" comment would be helpful. Absent documentation, the comment is unhelpful.